pytorch_lightning/loggers/comet.py

```diff
@@ -239,13 +247,42 @@ def save_dir(self) -> Optional[str]:
 
     @property
     def name(self) -> str:
-        return str(self.experiment.project_name)
+        # don't create an experiment if we don't have one
```

Comment:
```suggestion
        # Don't create an experiment if we don't have one
```
Just for consistency with other comments